modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/DownloadDelay.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.sdk.domain.snapshot
+
+trait DownloadDelay[F[_]] {
```

Comment:
Instead of saying where it will be used, say what it is. It's like imperative vs declarative programming. It's basically "PeerDiscoveryDelay". You can use it in Download of course, but you can also use it in different places.

Reply:
The trait has been renamed to `PeerDiscoveryDelay`.